server/database/database.js

```diff
@@ -1 +1,16 @@
-// mongodb?
\ No newline at end of file
+const mysql = require('mysql');
```

Comment:
Here's my local database setup for testing. I want to get everything up and running before refactoring to another DB and hosting it on EC2.